client/src/main/java/io/split/engine/experiments/RefreshableSplitFetcher.java

```diff
@@ -160,21 +185,34 @@ public void runWithoutExceptionHandling() throws InterruptedException {
                 if (split.status != Status.ACTIVE) {
                     // archive.
                     toRemove.add(split.name);
+                    if (split.trafficTypeName != null) {
```

Comment:
Fer, I need more context here to understand why you're marking a traffic type for removal based on the split's status